src/test/java/ru/hogwarts/school/controller/FacultyControllerMVCTest.java

```diff
@@ -0,0 +1,105 @@
+package ru.hogwarts.school.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import ru.hogwarts.school.model.Faculty;
+import ru.hogwarts.school.model.Student;
+import ru.hogwarts.school.repository.FacultyRepository;
+import ru.hogwarts.school.repository.StudentRepository;
+import ru.hogwarts.school.service.StudentService;
+
+import java.util.List;
+import java.util.Optional;
+
+@WebMvcTest(controllers = FacultyController.class)
+public class FacultyControllerMVCTest {
+    @MockBean
+    StudentRepository studentRepository;
+    @MockBean
+    FacultyRepository facultyRepository;
+    @SpyBean
+    StudentService studentService;
```

Comment:
```suggestion
    @SpyBean
    StudentServiceImpl studentService;
    @SpyBean
    FacultyServiceImpl facultyService;
```
Нужно поправить вот так.